lib/marklogic.js

```diff
@@ -727,7 +727,7 @@ MarkLogicClient.prototype.endpointCaller =function endpointCaller(endpointDeclar
 function initClient(client, inputParams) {
   var connectionParams = {};
   var isSSL = (inputParams.ssl == null) ? false : inputParams.ssl;
-  var keys = ['host', 'port', 'database', 'user', 'password', 'authType', 'token', 'basePath'];
+  var keys = ['host', 'port', 'database', 'user', 'password', 'authType', 'token', 'basePath','apikey','tenancyUrl'];
```

Comment:
As mentioned in our meeting, I think `host` is what a user would expect, as "tenancy URL" refers to the entire URL. 

Also, the Java Client is using "cloudApiKey" (and ml-gradle is using "mlCloudApiKey"). I think it's good to include the "cloud" prefix, as "API key" by itself is a little vague. And since "API" and "key" are two separate words, the "K" should be capitalized.